### ISSUE TITLE
Raspberry Pi

### ISSUE THREAD

**Issue by slashtechno** [MattMoony/figaro#12]
Does this work on Raspberry Pi. I would love to use this tool, it looks amazing.

**Comment by MattMoony** [comment]
Hey, sorry for the late reply!

Assuming your Raspberry Pi setup fulfils all requirements and you have some audio in-/output, then, yes - it should work just fine ^^!

Please, feel free to respond if you do run into any trouble while installing Figaro!

**Comment by slashtechno** [comment]
@MattMoony It is stuck on `reify:date-fns: timing reifyNode:node_modules/rxjs Completed in 62451ms
`

**Comment by MattMoony** [comment]
Hmm, did you install it using `setup.sh`? And if so, did everything finish successfully?

**Comment by slashtechno** [comment]
@MattMoony I get this error:
```
[2021-03-02~16:30|*] Installing node requirements ... 

up to date, audited 2354 packages in 20s

6 vulnerabilities (1 moderate, 5 high)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.

```

Could you please edit the script so it installs all packages that are needed?

**Comment by MattMoony** [comment]
Oh, that's alright, that's not really an error - it's just `npm` telling me that I need to update my dependencies again ... ^^
But... now that you have run this setup script, it still doesn't work?

**Comment by slashtechno** [comment]
@MattMoony I get this error when I run `python figaro.py`
```
Traceback (most recent call last):
  File "figaro.py", line 3, in <module>
    import pash.misc, pash.cmds, sys
ImportError: No module named pash.misc
```

**Comment by MattMoony** [comment]
What version of python are you using?

**Comment by slashtechno** [comment]
Python 3.7.3

**Comment by MattMoony** [comment]
Hm, you could try to install pash manually (`python3 -m pip install pash-cmd==0.0.5`), even though actually this should have already happened during setup ... :thinking:

**Comment by slashtechno** [comment]
Still does not work :(

**Comment by rjousse18** [comment]
same error, when i run with python and python3

**Comment by MattMoony** [comment]
I seem to have forgotten to update the `pash-cmd` after creating a new version to fix exactly this error ^^.
Please re-open the issue if there are more errors still.

**Comment by slashtechno** [comment]
I still get an error:
```
[2021-03-29~11:41|*] Installing node requirements ... 
npm WARN cleanup Failed to remove some directories [
npm WARN cleanup   [
npm WARN cleanup     '/home/pi/figaro/figaro/gui/web/node_modules',
npm WARN cleanup     [Error: ENOTEMPTY: directory not empty, rmdir '/home/pi/figaro/figaro/gui/web/node_modules/mozjpeg'] {
npm WARN cleanup       errno: -39,
npm WARN cleanup       code: 'ENOTEMPTY',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: '/home/pi/figaro/figaro/gui/web/node_modules/mozjpeg'
npm WARN cleanup     }
npm WARN cleanup   ]
npm WARN cleanup ]
npm notice 
npm notice New minor version of npm available! 7.6.3 -> 7.7.5
npm notice Changelog: https://github.com/npm/cli/releases/tag/v7.7.5
npm notice Run npm install -g npm@7.7.5 to update!
npm notice 
npm ERR! code 1
npm ERR! path /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg
npm ERR! command failed
npm ERR! command sh -c node lib/install.js
npm ERR! ⚠ Command failed: /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg -version
npm ERR! /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: 1: /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: ELF: not found
npm ERR! /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: 2: /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: �: not found
npm ERR! /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: 3: /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: �: not found
npm ERR! /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: 4: /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: �j�jI�: not found
npm ERR! /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: 5: /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: �j�j�@@DDP�tdxK: not found
npm ERR! /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: 6: /home/pi/figaro/figaro/gui/web/node_modules/mozjpeg/vendor/cjpeg: Syntax error: ")" unexpected
npm ERR! 
npm ERR! 
npm ERR!   ⚠ mozjpeg pre-build test failed
npm ERR!   ℹ compiling from source
npm ERR!   ✖ Error: Command failed: /bin/sh -c autoreconf -fiv
npm ERR! autoreconf: Entering directory `.'
npm ERR! autoreconf: configure.ac: not using Gettext
npm ERR! autoreconf: running: aclocal --force 
npm ERR! autoreconf: configure.ac: tracing
npm ERR! autoreconf: configure.ac: not using Libtool
npm ERR! autoreconf: running: /usr/bin/autoconf --force
npm ERR! configure.ac:23: error: possibly undefined macro: AC_PROG_LIBTOOL
npm ERR!       If this token and others are legitimate, please use m4_pattern_allow.
npm ERR!       See the Autoconf documentation.
npm ERR! autoreconf: /usr/bin/autoconf failed with exit status: 1
npm ERR! 
npm ERR! 
npm ERR!     at /home/pi/figaro/figaro/gui/web/node_modules/bin-build/node_modules/execa/index.js:231:11
npm ERR!     at runMicrotasks (<anonymous>)
npm ERR!     at processTicksAndRejections (node:internal/process/task_queues:94:5)
npm ERR!     at async Promise.all (index 0)

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2021-03-29T16_49_18_201Z-debug.log

```
